src/FixedLocalTupleList.py

```diff
@@ -0,0 +1,92 @@
+#  Copyright (C) 2017
+#      Max Planck Institute for Polymer Research
+#  
+#  This file is part of ESPResSo++.
+#  
+#  ESPResSo++ is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#  
+#  ESPResSo++ is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#  
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+
+
+r"""
+*****************************
+**espressopp.FixedLocalTupleList**
+*****************************
+This class can contain many tuple which store a arbitrary positive number, which should be more than 2, of local (real + ghost) particle id.
+
+For using this class, there are 3 conditions.
+
+1. The length of all tuple must be same.
+2. Particles in one tuple must be in a same or neighbor cell list.
+3. Particle id should not be stored redundantly.
+
+These conditions are available only for the identical object.
```

Comment:
Ok, so maybe something like that:

> The length of the tuples on the list is a local property of the object, hence different FixedLocalTupleList objects can handle tuples of different lengths.

btw, how difficult would be to make it general so this length condition is not required?